src/content/changes/highlights/2023-06-29.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-06-29T14:00:00
+title: Highlights for the week ending June 29 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
+- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.9](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0149---2023-06-26) makes the policy-reporter scrape timeout and interval configurable to allow more collection time in clusters with many policies.
+- [security-bundle](https://github.com/giantswarm/security-bundle) version [1.0.1](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#101---2023-06-26) is the first release intended to be installed by default on Kubernetes 1.25 and later. It disables PSPs in favour of enforced Kyverno PSS-restricted policies. New platform releases are shipped with this new version by default. We will inform you in the platform release notes about the change and steps you need to take to upgrade. [More info about the change can be read here](https://github.com/giantswarm/security-bundle/tree/main#installing). 
```

Comment:
```suggestion
- [security-bundle](https://github.com/giantswarm/security-bundle) version [1.0.1](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#101---2023-06-26) is the first release intended to be installed by default on Kubernetes 1.25 and later. It disables PSPs in favour of enforced Kyverno PSS restricted policies. New platform releases are shipped with this new version by default. We will inform you in the platform release notes about the change and steps you need to take to upgrade, as it may require removing or modifying Kyverno or Security Bundle apps prior to upgrade [More info about the change can be read here](https://github.com/giantswarm/security-bundle/tree/main#installing). 
```